dan_layer/core/src/workers/pacemaker_worker.rs

```diff
@@ -287,6 +296,37 @@ mod tests {
             .is_err());
     }
 
+    #[tokio::test]
+    async fn test_remove_pending_timeouts() {
+        let (_, rx_signal) = channel::<PacemakerTimer<u8>>(10);
+        let (tx_waiter_status, _) = channel::<u8>(10);
+
+        let mut pacemaker = Box::new(Pacemaker::new(rx_signal, tx_waiter_status));
```

Comment:
```suggestion
        let mut pacemaker = Pacemaker::new(rx_signal, tx_waiter_status);
```

Reply:
Ups, this was left from our previous conversation. Addressed 👍